packages/docusaurus-theme-classic/src/theme/LastUpdated/index.tsx

```diff
@@ -8,23 +8,37 @@
 import React from 'react';
 import Translate from '@docusaurus/Translate';
 import {ThemeClassNames} from '@docusaurus/theme-common';
+import {useDateTimeFormat} from '@docusaurus/theme-common/internal';
 import type {Props} from '@theme/LastUpdated';
 
 function LastUpdatedAtDate({
   lastUpdatedAt,
-  formattedLastUpdatedAt,
 }: {
   lastUpdatedAt: number;
-  formattedLastUpdatedAt: string;
 }): JSX.Element {
+  const convertedLastUpdateAt = lastUpdatedAt * 1000;
+
+  const dateTimeFormat = useDateTimeFormat({
+    day: 'numeric',
+    month: 'short',
+    year: 'numeric',
+    timeZone: 'UTC',
+  });
+
+  const formatLastUpdatedAt = (lastUpdated: number) =>
+    dateTimeFormat.format(new Date(lastUpdated));
+
+  const formattedLastUpdatedAt =
+    lastUpdatedAt && formatLastUpdatedAt(convertedLastUpdateAt);
```

Comment:
`lastUpdatedAt` is always defined here, you can remove `formattedLastUpdatedAt` and just call `dtf.format(date)` in component?